beets/ui/commands.py

```diff
@@ -801,6 +830,53 @@ def should_resume(self, path):
                            "was interrupted. Resume (Y/n)?"
                            .format(displayable_path(path)))
 
+    def _get_plugin_options(self, task):
```

Comment:
No worries, I'll update the name!

Also, I kind of defaulted to using "*choice*" for the concept of "*an item the user can select during the prompt*", but it might be a language issue. If you feel that it is more natural to use "*option*" for that concept, please let me know so I can make a pass at the code and documentation for trying to make everything more consistent.